data_tools/buildhub/utils.py

```diff
@@ -0,0 +1,93 @@
+from os.path import basename
+
+
+def build_record_from_url(url):
+    record = {}
+    parts = basename(url).split('.')
+    url_parts = url.split('/')
+
+    product_name, major_version = parts[0].split('-')
+    channel = 'release'
+
+    if 'nightly' in url:
+        version = '{}.{}'.format(major_version, parts[1])
+        locale = parts[2]
+        platform = parts[3]
+        if 'aurora' in url:
```

Comment:
It happens for the Aurora URLs (there is a test covering it.

https://github.com/mozilla-services/buildhub/pull/30/commits/63918d49b6035c7b2af3071571d9ea15b55d280d